rfcs/rfc-009-staking.md

```diff
@@ -0,0 +1,59 @@
+
+I.e. Each vote has an associated lockout time (PoH duration) that represents a duration by any additional vote from that validator must be in a PoH that contains the original vote, or a portion of that validator's stake is slashable. This duration time is a function of the initial vote PoH count and all additional vote PoH counts.  It will likely take the form:
+
+Lockout<sub>i</sub>(PoH<sub>i</sub>, PoH<sub>j</sub>) = PoH<sub>j</sub> + K * exp((PoH<sub>j</sub> - PoH<sub>i</sub>) / K)
+
+Where PoH<sub>i</sub> is the height of the vote that the lockout is to be applied to and PoH<sub>j</sub> is the height of the current vote on the same branch. If the validator submits a vote on a different PoH branch on any PoH<sub>k</sub> where k > j > i and PoH<sub>k</sub> < Lockout(PoH<sub>i</sub>, PoH<sub>j</sub>), then a portion of that validator's stake is at risk of being slashed.
+
+The exponential nature of the lockout time protects against the ability for adversarial nodes, that may have the ability to generate faster (against clock time) PoH streams, from creating partitions and censoring nodes. Due to the highly optimized hardward properties of the `sha256` hash function, a faster PoH stream from a validator isn't likely to be significant and their PoH stream will increase in difference only linearly. Additionally, leader rotation times reduces the global speedup available to a malicious node to the fraction speed-up they've achieved multiplied by the % of stake they have in the total deposit (i.e. the % of time they are elected leader).
+
+It is likely that a reward will be offered as a % of the slashed amount to any node that submits evidence of this slashing condition being violated to the PoH.
+
+#### Partial Slashing
+In the schema described so far, when a validator votes on a given PoH stream, they are committing themselves to that branch for a time determined by the vote lockout. An open question is whether validators will be hesitant to begin voting on an available branch if the penalties are perceived too harsh for an honest mistake or flipped bit.
+
+One way to address this concern would be a partial slashing design that results in a slashable amount as a function of either a) the fraction of validators, out of the total validator pool, that were also slashed during the same time period (ala Casper), b) the amount of time since the vote was cast (e.g. a linearly increasing % of total deposited as slashable amount over time), or both.  This is an area currently under exploration
```

Comment:
I'd find this easier to read if you used markdown's numbered list here instead of "a) b)" in one long line.